xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -3024,15 +3024,22 @@ bool CFFmpegPostproc::AddPicture(CVaapiDecodedPicture &inPic)
 
   av_frame_get_buffer(m_pFilterFrameIn, 64);
 
+#if defined(SSE4_1_FOUND)
   uint8_t *src, *dst;
+#else
+  uint8_t __attribute__((unused)) *src, *dst;
```

Comment:
Why declare the variable, if it is unused with disabled SSE4.1, instead of moving the uses of it inside the (SSE4.1) if def?